[v1.0.0b] - Server error / Cannot find module 'vuetify/src/locale'

### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

With yesterday's nightly build the web-frontend does not load with "Server error - An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details."

Frontend log:

```
 ERROR  Cannot find module 'vuetify/src/locale' from '/app'

  at Function.resolveSync [as sync] (node_modules/resolve/lib/sync.js:102:15)
  at node_modules/vue-server-renderer/build.prod.js:1:77905
  at Object.<anonymous> (webpack:/external "vuetify/src/locale":1:0)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Module.<anonymous> (server.js:26791:15)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Object.<anonymous> (server.js:22380:18)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at server.js:118:18
  at Object.<anonymous> (server.js:121:10)

```

### Deployment

Docker (Linux)

### Deployment Details

Version: v1.0.0beta-5
Build: 5dc253799d1d001858dbfd59a6376378d2ca41fc

--- a/frontend/vuetify.options.js
+++ b/frontend/vuetify.options.js
@@ -1,8 +1,11 @@
-import * as locale from "vuetify/src/locale";
+import * as locale from "vuetify/lib/locale";
+
+// Ensure not null or undefined
+if (!locale || !locale.en) {
+  throw new Error("Vuetify locale not found");
+}
 
 export default {
-  breakpoint: {},
-  icons: {},
   lang: {
     locales: {
       "el-GR": locale.el,

